### PR TITLE
Corrects an issue where updater would not pass service apikeys during service registration

### PR DESCRIPTION
**Pull request by keNEticHEx**
Corrects an issue where service apikeys were not being passed from the updater service to the service server during service registration. This resulted in the update failing unless the default apikey was used.

### CODE REVIEW

**Comment by keNEticHEx** [comment]
Has a maintainer had a chance to review this? The updater service remains broken on the current releases until this, or a similar solution can be merged.

**Comment by cccs-douglass** [comment]
Something related to this got caught up in a bigger set of changes, which will be delayed. So for now, merging this alone seems worthwhile. Thanks for the help.

**Review by cccs-douglass — APPROVED** [review]

**Pull request merged**
